opal/corelib/module.rb

```diff
@@ -395,14 +396,27 @@ def define_method(name, method = undefined, &block)
               end
 
     %x{
-      var id = '$' + name;
+      // Wrapping and forwarding is required to make super work, as the
+      // same block can be used to define multiple methods.
+      var wrapper = function() {
+        block.$$jsid        = name
+        block.$$p           = wrapper.$$p
+        block.$$s           = null;
+        block.$$def         = wrapper;
+        block.$$define_meth = true;
+
+        return block.apply(this, arguments)
+      }
+
+      var id = `$${name}`
```

Comment:
This is another thing I haven't noticed before. I would rather if it was '$'+name...

Reply:
Also, the linter has noticed it, but didn't fail the job:

```
* Parsing error: Unexpected character '`'
  - Compiled: tmp/lint/opal.js:3511:16
  - Original: corelib/module.rb:411:0
```

Reply:
sorry did that out of habit, will fix it along with the un-failed job 👍